moped-editor/src/utils/dataGridHelpers.js

```diff
@@ -0,0 +1 @@
+export const defaultEditColumnIconStyle = { fontSize: "24px" };
```

Comment:
thanks for pulling this out as a variable since we need to override the default size of the DataGridPro action buttons to the ["medium" font size](https://mui.com/material-ui/api/icon/#props) that the non-`GridActionsCellItem` wrapped icons default to already. I think this size is more in line with what we saw in Material Table which is good. These PRs are teaching me so much about the details of MUI! 🍄

I am starting to wonder if we should start moving towards `rem` in our custom sizes like we have with `theme.spacing` but I know that we have a lot of ground to cover with the code written already. I'll keep it in mind.